packages/coil-extension/src/popup/Index.tsx

```diff
@@ -7,6 +7,7 @@ import { PopupHost } from './types'
 import { PopupHostContext } from './context/popupHostContext'
 import { StoreContext, useStoreState } from './context/storeContext'
 import { ViewRouter } from './components/views/ViewRouter'
+import { RouterProvider } from './context/routerContext'
```

Comment:
Should we update the extension to material ui 5 while doing this? 
Might be a good chance ?

Reply:
That's a good thought, I don't think there are any breaking changes but I'll check the docs before updating. If it's all good then an update will be a separate PR.